utils/src/Vulkan/Utils/FromGL.hs

```diff
@@ -0,0 +1,276 @@
+{-# LANGUAGE CPP #-}
+
+{-
+  GL-to-vulkan conversion from
+  https://github.com/KhronosGroup/KTX-Software/blob/bf849b7f/lib/vk_format.h
+-}
+
+{-
+Copyright (c) 2016 Oculus VR, LLC.
```

Comment:
Possible to replace this with a BSD-3-clause licensed file? Or otherwise mention in LICENSE that some files may be explicitly under a different license.

Surprises me that this file is even copyrightable (by someone not Khronos..

Reply:
The copyright is inalienable and orthogonal to license terms. It's basically "we did this" authorship mark. Alternative may be BSD3, but still has to state the author.